### DISCUSSION TITLE
Welcome to winmerge Discussions!

### DISCUSSION THREAD

**Discussion by sdottaka** [WinMerge/winmerge#502] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gitoffthelawn** [comment]
Looking forward to helping build this community and continuing to improve WinMerge.

**Reply by JackKumar33** [reply]
Hi all While I try to compare two .xls file I am getting the below shown error. Could any one help me
![Screenshot_986](https://user-images.githubusercontent.com/75119289/103760521-3b1d4a80-503b-11eb-8303-74de7ed8cffc.png)

**Comment by Gitoffthelawn** [comment]
@JackKumar33 Hi Jack. You'll want to post your question as a new discussion item (instead of in this thread).  Don't forget to include details such as which OS you are using and as well as your version of WinMerge.
